app/editor_ui/views/project_views.py

```diff
@@ -0,0 +1,77 @@
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.db.models import (
+    Count,
+    Q,
+)
+from django.urls import reverse
+from django.views.generic import DetailView, ListView
+
+from app.editor_ui.forms import (
+    ProjectForm,
+)
+from app.editor_ui.mixins import CreatedByUserMixin, SuperuserRequiredMixin
+from app.editor_ui.views.base_views import BaseCreateView
+from app.projects.models import Project
+
+
+class ProjectCreateView(
+    CreatedByUserMixin,
+    SuperuserRequiredMixin,
+    LoginRequiredMixin,
+    BaseCreateView,
+):
+    model = Project
+    form_class = ProjectForm
+    object_name = "Project"
```

Comment:
Before the change, the project create view seems to be using a custom template: 
https://github.com/nationalarchives/ds-feedback/pull/43/files#diff-efc44195ff7015386ab8647c971a1baaf50d7a63c56cc5cf7578139f6cf52184L42 and not `editor_ui/create.html`. 

Should that be changed here? If not, perhaps we can remove the unused template.